qutip/solver/parallel.py

```diff
@@ -70,7 +71,9 @@ def serial_map(task, values, task_args=None, task_kwargs=None,
         The optional additional keyword argument to the ``task`` function.
     reduce_func : func (optional)
         If provided, it will be called with the output of each tasks instead of
-        storing a them in a list.
+        storing a them in a list. It should return None or a number.
+        When returning a number, it represent the estimation of the number of
+        task left. On a return >= 0, the map will end early.
```

Comment:
```suggestion
        task left. On a return <= 0, the map will end early.
```
I think the docstring had this the wrong way around.